src/components/addComments.vue

```diff
@@ -0,0 +1,175 @@
+<template>
+  <section className='add_message_section'>
```

Comment:
```suggestion
  <section class='add_message_section'>
```
vue understand all html attributes. fix here and everywhere else